deploy/kubernetes/rbac.yaml

```diff
@@ -32,9 +32,13 @@ rules:
   - apiGroups: [""]
     resources: ["events"]
     verbs: ["list", "watch", "create", "update", "patch"]
-  - apiGroups: [""]
-    resources: ["secrets"]
-    verbs: ["get", "list"]
+# Secret permission is optional.
```

Comment:
I think you need to add space and make sure this aligns with - apiGroups: [""] on line 32.  yaml file is space sensitive.

Reply:
Actually, no need to worry about that. Comment lines in yaml file will be ignored. I have test it and it can apply/create. Maybe some CO have different behavior, so I also update the yaml file as you recommend